lib/xclarity_client/services/storage_management.rb

```diff
@@ -0,0 +1,12 @@
+module XClarityClient
+  #
+  # This class provides a set of services related to Storage endpoint
+  #
+  class StorageManagement < Services::XClarityService
+    manages_endpoint Storage
+
+    def population(opts = {})
```

Comment:
__Required:__
Please, avoid using `population` method. You can just use the `fetch_all` defined on `Services::XClarityService`.

This way we remove some code duplication. [More details here](https://github.com/lenovo/xclarity_client/pull/98).

Reply:
True, I didin't noticed that. Updated code to use fetch_all instead.